views/editor.dt

```diff
@@ -19,20 +19,20 @@ block content
 		div(style="padding-left: 0px; padding-right: 0px")
 			div#command-box.text-right
 				button.btn.btn-primary(ng-click="run()")
-					i.fa.fa-play(aria-hidden="true")
 					span Run
-				button.btn.btn-primary(ng-click="format()")
 					i.fa.fa-play(aria-hidden="true")
+				button.btn.btn-primary(ng-click="format()")
 					span Format
+					i.fa.fa-play(aria-hidden="true")
```

Comment:
I'd like to find a good alternative to reusing the play icon. I opened a separate issue for that: #572. (We should leave it for a separate PR, of course.)